data/tools/GUI.pyw

```diff
@@ -10,11 +10,13 @@
 
 # threading and subprocess are needed to run wmllint without freezing the window
 # codecs is used to save files as UTF8
+# gettext provides internationalization and localization (i18n, l10n)
```

Comment:
Why is the localization stuff from PR #7627 in this one too?

Reply:
Sorry, it seems I rebased against the wrong branch.